lib/einhorn/prctl.rb

```diff
@@ -0,0 +1,26 @@
+module Einhorn
+  class PrctlAbstract
+    def self.get_pdeathsig
+      raise NotImplementedError
+    end
+
+    def self.set_pdeathsig(signal)
+      raise NotImplementedError
+    end
+  end
+
+  class PrctlUnimplemented < PrctlAbstract
+    # Deliberately empty; NotImplementedError is intended
+  end
```

Comment:
I wasn't happy with any way I could write this.

- Using `PrctlAbstract` as a real class, not an abstract one, feels bad.
- Not having a note in the class hierarchy mentioning that both `PrctlLinux` and `PrctlUnimplemented` must implement the same interface also feels bad. (This is probably the weakest argument)
- Inheriting from `PrctlUnimplemented` and then making it not-unimplemented also feels bad.

This hierarchy is me trying to avoid all those bad feels, though it ended up looking verbose. Overall though, I prefer it; the other option that doesn't feel horrible would be to not have `PrctlAbstract` at all, and just have `PrctlLinux` and `PrctlUnimplemented` standalone.